src/main/java/subway/request/LineRequest.java

```diff
@@ -1,4 +1,6 @@
-package subway.line;
+package subway.request;
```

Comment:
request 와 reponse 는 Controller 레벨에만 사용하게 되니 
controller 패키지 밑으로 들어가도 될 것 같아요
```suggestion
package subway.controller.request;
```